container.go

```diff
@@ -93,6 +93,7 @@ type FromDockerfile struct {
 	Tag            string                         // the tag label for image, defaults to UUID
 	BuildArgs      map[string]*string             // enable user to pass build args to docker daemon
 	PrintBuildLog  bool                           // enable user to print build log
+	BuildLogWriter io.Writer                      // for output of build log if PrintBuildLog is true, defaults to os.Stderr
```

Comment:
question: should the default be discard as it can become quite spammy for large builds?

Reply:
Refers to the discussion about default as discard above